virttest/storage.py

```diff
@@ -83,6 +84,9 @@ def file_exists(params, filename_path):
         keyring_conf = params.get("image_ceph_keyring_conf")
         return ceph.rbd_image_exist(ceph_monitor, rbd_pool_name,
                                     rbd_image_name, ceph_conf, keyring_conf)
+
+    if params.get('enable_nvme') == 'yes':
+        return nvme.file_exists(params, filename_path)
```

Comment:
storage_type check here?
if we have another nvme backend, e.g. [nvme], then enable_nvme = yes, but the storage_type(nvme vs. nvme-direct) is different, and this file_exits only check the nvme-direct device.

Reply:
Actual, nvme is PCI device , if you didn't unbind it from the host , it will be a block device, like other common block devices. so this scenario, you don't need to new a storage backend, just treat it as a local device. like sda, sdb.
```
[root@dell-per740xd-01 kar]# lsblk
NAME                             MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda                                8:0    0   372G  0 disk
├─sda1                             8:1    0     1G  0 part /boot
└─sda2                             8:2    0   371G  0 part
  ├─rhel_dell--per740xd--01-root 253:0    0    50G  0 lvm  /
  ├─rhel_dell--per740xd--01-swap 253:1    0  31.4G  0 lvm  [SWAP]
  └─rhel_dell--per740xd--01-home 253:2    0 289.6G  0 lvm  /home
sdb                                8:16   0 558.4G  0 disk
└─sdb1                             8:17   0 476.9G  0 part
sdc                                8:32   0 558.4G  0 disk
nvme0n1                          259:0    0 745.2G  0 disk
```
That is one way to use the nvme.
The other way is to unbind it from the host, let it independent as a new storage backend that is the nvme direct method.